src/test/java/unit/MountControllerTest.java

```diff
@@ -0,0 +1,171 @@
+package unit;
+
+import static org.mockito.Mockito.*;
```

Comment:
For Terra repos, we avoid using * imports for clarity. If intellij did that automatically, there's a setting to stop it from doing so (something like "Class count to use import with '*'" to 99